src/HackF5.UnitySpy/Detail/MemoryObject.cs

```diff
@@ -1,31 +1,40 @@
-﻿namespace HackF5.UnitySpy.Detail
+﻿using System;
```

Comment:
Nitpicking: throughout the project, Brian was putting the imports inside the `namespace` declaration, so maybe we can keep things that way for consistency?

Reply:
Done